src/api/helpers/is-valid-postcode/index.ts

```diff
@@ -0,0 +1,8 @@
+import { postcodeValidator } from 'postcode-validator';
+
+/**
+  Validates if a value is a valid postcode
+ * @param {String} postcode - the value to validate
+ * @returns Boolean - true if valid, false if not
+ */
+export const isValidPostcode = (postcode: string): boolean => postcodeValidator(postcode, 'GB');
```

Comment:
Not sure if we need to introduce a new dependency, can not use `joi` since it already being used?
https://github.com/alanshaw/joi-postcode

Reply:
we use this in the UI and it works perfectly which is why I moved it across to the API